.github/workflows/ci.yaml

```
@@ -0,0 +1,26 @@
+          version: nightly
+      - name: Install forge-std
+        run: |
+          forge install foundry-rs/forge-std --no-commit
+          forge remappings > remappings.txt
+      - name: Run non-Kontrol tests
+        run: |
+          forge test --no-match-path "src/test/kontrol/*" -vvv
+      - name: Run snapshot without Kontrol tests
+        run: forge snapshot --no-match-path "src/test/kontrol/*"
+
+  
```

yamllint (1.29.0-1)

[error] no new line character at the end of file

(new-line-at-end-of-file)